### PR TITLE
Added task description

### PR DESCRIPTION
**Pull request by danylott**

### CODE REVIEW

**Review by Abnormaltype — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+    "c15f56a2a392c950524f499093b78266427d21291b7d7f9d94a09b4e41d65628",
+    "4cd1a028a60f85a1b94f918adb7fb528d7429111c52bb2aa2874ed054a5584dd",
+    "40900aa1d900bee58178ae4a738c6952cb7b3467ce9fde0c3efa30a3bde1b5e2",
+    "5e6bc66ee1d2af7eb3aad546e9c0f79ab4b4ffb04a1bc425a80e6a4b0f055c2e",
+    "1273682fa19625ccedbe2de2817ba54dbb7894b7cefb08578826efad492f51c9",
+    "7e8f0ada0a03cbee48a0883d549967647b3fca6efeb0a149242f19e4b68d53d6",
+    "e5f3ff26aa8075ce7513552a9af1882b4fbc2a47a3525000f6eb887ab9622207",
+]
+```
+Of course, your first thought was to look through rainbow tables and test your luck.
```
Comment (alenatovstukha) [inline README.md:21]: Of course, your first thought was to look through rainbow tables and test your luck. But you visited that website and understand that its password field accepts only numeric symbols from `0` to `9`. There is a limitation — the password length always must be `8` symbols (extremely weak security — you must agree). Also, it seems that this website doesn't use any additional security while hashing their passwords (like salting).

File: README.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+But you visited that website, and understand, that its password field accepts only numeric symbols from `0` to `9`, and
+also there is a limitation, that password always must be 8 symbols length (extremely weak security - you must agree). 
+Also, it seems, that this website doesn't use any additional security while hashing their passwords (like salting).
+
+So, you as a hacker already got a point, that you can easily brute force all possible variants of passwords,
+and check the corresponding hashes using this simple `sha256_hash_str` function:
+```python
+import sha256
+
```
Comment (alenatovstukha) [inline README.md:30]: тут нужно две пустые строки?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+    "7e8f0ada0a03cbee48a0883d549967647b3fca6efeb0a149242f19e4b68d53d6",
+    "e5f3ff26aa8075ce7513552a9af1882b4fbc2a47a3525000f6eb887ab9622207",
+]
+```
+Of course, your first thought was to look through rainbow tables and test your luck.
+But you visited that website, and understand, that its password field accepts only numeric symbols from `0` to `9`, and
+also there is a limitation, that password always must be 8 symbols length (extremely weak security - you must agree). 
+Also, it seems, that this website doesn't use any additional security while hashing their passwords (like salting).
+
+So, you as a hacker already got a point, that you can easily brute force all possible variants of passwords,
```
Comment (alenatovstukha) [inline README.md:26]: So, you as a hacker already got the point that you can easily brute force all possible variants of passwords and check the corresponding hashes using the `sha256_hash_str` function:

File: README.md [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+and check the corresponding hashes using this simple `sha256_hash_str` function:
+```python
+import sha256
+
+
+def sha256_hash_str(to_hash: str) -> str:
+    return sha256(to_hash.encode("utf-8")).hexdigest()
+```
+
+And yes, your task is to find these 10 passwords (8-numeric symbols strings), and print them to console.
```
Comment (alenatovstukha) [inline README.md:36]: Your task is to find these `10` passwords (`8`-numeric symbols strings) and print them to the console.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+
+
+Notes:
```
Comment (alenatovstukha) [inline README.md:39]: **Please note:**
- you should find `10` passwords (not `9`) — сheck it;
- you should use setup to speed up this brute forcing as much as possible;
- attach the screenshot of your script results (the console) to the PR.

**Review by alenatovstukha — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (danylott) [reply README.md:30]: Yes :) I'ts Python code style

File: README.md (hunk 5 above)
Comment (danylott) [reply README.md:39]: Thank you, all resolved

**Pull request merged**